collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FieldListUpdateTest.java

```diff
@@ -391,6 +391,18 @@ public void searchMinimalInFieldList() {
                 .assertSelectMinimalDialogAnswer("Strawberries");
     }
 
+    @Test
+    public void listOfQuestionsShouldNotBeScrolledToTheLastEditedQuestionAfterUpdatingTheLayout() {
```

Comment:
I'm not sure about the name here - it kind of describes what's happening "under the hood" rather than the behaviour. Could it be `listOfQuestionsShouldNotBeScrolledToTheLastEditedQuestionAfterClickingOnAQuestion`?